containers/debian/12.7/Containerfile.upstream

```diff
@@ -0,0 +1,10 @@
+#
+# A Debian image tailored for tmt test suite
+#
+# tmt/tests/debian/12.7/upstream:latest
+#
+
+FROM docker.io/library/debian:12.7
+
+    # Populate apt cache
```

Comment:
weird ident

Reply:
This is expected, it's aligned with the command it describes, not with `RUN`.

Reply:
Interesting, seems weird to me as well. Ale proti gustu... ;-)

Reply:
It's unusual, and it's caused by that `RUN` command. It starts to be less distracting once there are more commands:

```dockerfile
    # Populate dnf cache
RUN    dnf makecache \
    # Create unprivileged user and setup sudo for it
    && dnf install -y /usr/sbin/useradd \
    && useradd fedora \
    && usermod -aG wheel fedora \
    && echo -e 'fedora\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
```

Maybe we could put `RUN` on its own line, and help the visual impression a bit by changing it to this?

```dockerfile
RUN \
    # Populate dnf cache
    && dnf makecache \
    # Create unprivileged user and setup sudo for it
    && dnf install -y /usr/sbin/useradd \
    && useradd fedora \
    && usermod -aG wheel fedora \
    && echo -e 'fedora\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
```

Reply:
No problemo, it's fine. We can improve if/when there are more lines.